main.py

```diff
@@ -165,6 +165,8 @@ def dict_unique(song_list, trackname_convention):
         trackname = f"{song.title} - {song.artists}"
         if trackname_convention == 2:
             trackname = f"{song.artists} - {song.title}"
+        if len(trackname) > 260: # Added because you can't have filenames with more that 260 chars
+            trackname = song.title 
```

Comment:
```suggestion
trackname = song.title[:255] + "..." # If the author makes the trackname longer that 260 chars, then it just returns the title, otherwise, if the title is more then 260 chars itself, then it slices it.
```